core/src/main/kotlin/com/gitlab/kordlib/core/entity/Member.kt

```diff
@@ -90,17 +90,13 @@ class Member(
     suspend fun getPermissions(): Permissions {
         val guild = getGuild()
         val owner = guild.ownerId == this.id
-        if (owner) return Permissions {
-            +Permission.All
-        }
+        if (owner) return Permissions(Permission.All)
 
         val everyone = guild.getEveryoneRole().permissions
         val roles = roles.map { it.permissions }.toList()
 
-        return Permissions {
-            +everyone
-            roles.forEach { +it }
-        }
+        return roles.fold(everyone) { acc, permissions -> acc + permissions }
```

Comment:
This is quite a bit less efficient than the previous version. Obviously this won't lead to any noticeable differences but I think it's a good argument in favour of not removing the builder DSL.